src/main/webapp/browserslist

```diff
@@ -0,0 +1,12 @@
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
```

Comment:
observation: i usually see this within the package.json file. I'm sure its fine here.

Reply:
A **lot** of this was autogenerated, FYI. Though I will try to address your specific concerns sometime.